extism.go

```diff
@@ -542,6 +573,11 @@ func (plugin *Plugin) Call(name string, data []byte) (uint32, []byte, error) {
 
 	res, err := f.Call(ctx)
 
+	// TODO: Should we finish the trace now, or defer it?
+	if plugin.TraceCtx != nil {
+		defer plugin.TraceCtx.Finish()
```

Comment:
Decided to defer it because I wanted it to reflect the whole call, even though the kernel is not instrumented (as far as I know), so it probably doesn't matter anyway. However, since we are wrapping all of the kernel calls, we can use the manual instrumentation